### PR TITLE
update deprecated IPython.kernel import

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: jupyter_console/tests/test_image_handler.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 except ImportError:
     from mock import patch
 
-from IPython.kernel import KernelClient
+from ipykernel import KernelClient
```
Comment (takluyver) [inline jupyter_console/tests/test_image_handler.py]: We don't expose `KernelClient` at the top level, so this needs to be `from ipykernel.client`

Comment (minrk) [inline jupyter_console/tests/test_image_handler.py]: That's surprising and annoying.

Comment (minrk) [inline jupyter_console/tests/test_image_handler.py]: oh, duh, I meant jupyter_client.

Comment (takluyver) [inline jupyter_console/tests/test_image_handler.py]: Duh for me too for failing to spot that ;-)

**Pull request merged**